### ISSUE TITLE
Update website feature tour

### ISSUE THREAD

**Issue by bluesabre** [Xubuntu/xubuntu-website#11]
https://xubuntu.org/tour/

**Comment by philipzae** [comment]
What part did you want to update? Text, design, etc.

**Comment by bluesabre** [comment]
This came from the previous blueprint. @knomepasi @pleia2 Any ideas what changes we were going for?

**Comment by knomepasi** [comment]
I believe this was planned to be a general overhaul of the feature tour.

**Comment by philipzae** [comment]
Here are some things I think should be on the page.
1. Info about Xfce
2. Info about the default apps
3. Various panel arrangements
4. Possibly info about where it can be run (desktop, raspberry pi, WSL)

I think Ubuntu Mate does a good job with their features page https://ubuntu-mate.org/features/

**Comment by knomepasi** [comment]
One thing I find a bit suboptimal in the Mate Feature Tour is that its default "slide" is a list of application names/icons, of which many are not useful at all for new users. Those who are more into Linux and want a more thorough comparison of distros seem to be the target group there. Seems like the same target group is in their minds when they write about "traditional [desktop] metaphors" and "The legacy of GNOME 2".

I would want the Xubuntu Feature Tour to be written in a way that my mum can understand it. (We can surely do all kinds of geek-targeted lists too, but they shouldn't be the main concern.)

Otherwise I think we could definitely monkey ideas from them (or many other Feature Tours really). For example, it would probably make sense to have more screenshots.

Your list of things to be on the page I also agree with, but I would add the layer where I think Xubuntu excels the most: we have the Ubuntu base with great hardware support and the Xubuntu desktop built upon it delivers a stable, clean and easy to use user experience which can be easily customized with the power of Xfce.

**Comment by philipzae** [comment]
Yes I totally agree that the main slide page simply having the icons and names isn't optimal for the new user who doesn't already know what those apps are. [Kubuntu](https://kubuntu.org/feature-tour/) does a better job there, with a title, screenshot and description, though filling the entire page like they do seems a waste.

Yes of course we'd have to make it our own and make sure its as user-friendly that our moms could easily understand it. (I plan to test it on mine once we finish :D)

How would you like to collaborate on this? If you are fine with Google Docs and Zoom, we can start jotting things down in the google doc below and have a meeting to discuss it further. We should like also work on the front page, about page, and a few others, so that we decide what goes on each so there isn't any duplication.

https://docs.google.com/document/d/1vufKMXsXoz6fnl0oo_PEJg68ztLptSdj3Az3eZZAVrY/edit
